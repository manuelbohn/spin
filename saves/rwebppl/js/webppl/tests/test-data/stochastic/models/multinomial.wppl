var model = function() {
  return sum(map(function(s) { return s[0] * s[1] }, _.zip(multinomial([0.4, 0.5, 0.1], 5), [1, 2, 3])));
};
