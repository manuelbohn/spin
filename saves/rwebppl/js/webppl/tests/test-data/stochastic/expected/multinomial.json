{
  "mean": 8.5,
  "std": 1.431782
}